documentation/python/README.md

```diff
@@ -715,3 +715,131 @@ The following file illustrates a possible JJB configuration
             sonar.exclusions=tests/**/*.py
             sonar.ws.timeout=180
 ```
+#Code Coverage With Integration Tests
+
+Reference Docs:
+
+ - [How to generate coverage report for http based integration tests?](https://stackoverflow.com/questions/50689940/how-to-generate-coverage-report-for-http-based-integration-tests)
```

Comment:
For keeping 80 chars limit you can put the reference at the buttom:
```
[How to generate coverage report for http based integration tests?]
...
[How to generate coverage report for http based integration tests?]: https://stackoverflow.com/questions/50689940/how-to-generate-coverage-report-for-http-based-integration-tests
```

Reply:
done